newIDE/app/src/GameDashboard/UseGamesList.js

```diff
@@ -31,6 +31,14 @@ const useGamesList = (): GamesList => {
     null
   );
 
+  React.useEffect(
+    () => {
+      // Reset games when user logs out.
```

Comment:
When I'm logging out, I don't see my games any more 🤔 why is this needed?

Reply:
I was in a case, when the games were still available. I will try to reproduce it

Reply:
I don't reproduce it, I'll get rid of it